app/views/layouts/application.haml

```diff
@@ -111,6 +111,10 @@
             - if logged_in?
               %li= link_to 'Unread', unread_posts_path
               %li= link_to 'Replies Owed', owed_posts_path
+              - if current_user.replies_owed_indicator
+                - $owed = get_owed
```

Comment:
Don't need the $ here; that makes it a global, which we don't especially need it to be doing.